### ISSUE TITLE
Many, many lighting bugs.

### ISSUE THREAD

**Issue by TrademarkTM** [boy0001/FastAsyncWorldedit#233]
Hey!
I was using FAWE, but I had a big problem. There's SO MANY lighting bugs. The map gets entirely glitched, there's no light under trees, and after removing glowstone or any other light source it remains bright.

- http://prntscr.com/c4ilwn - there's no light sources there
- Under trees everything gets so dark, even when it's sunny. 
- In a flat world: http://prntscr.com/c4ir15

Information:
> Async light is OFF
> Lighting mode is 2
> Random light updates is false

* And NO, I will not use /fixlighting. It generates chunks, and I won't select my entire world to fix light.

* This is AFTER removing the plugin. Before this, there was 10x more glitches. And I can't fix them anymore. And yeah, it also generates glitching chunks. It's back to normal without the plugin.

**Comment by RoboMWM** [comment]
> Random light updates is false

Enabling this would be the way to periodically fix your lighting issues.

> And NO, I will not use /fixlighting.

Because this plugin uses NMS to paste things in directly, it will skip by bukkit's lighting stuff as well. This is provided so you can correct any lighting issues afterwards. Or just enable the random lighting updates as I mentioned prior.

**Comment by TrademarkTM** [comment]
I enabled random light updates, and it stays the same. And as I saw in other forums, it's not good when there's many players online.

So with/without random lighting the glitch occours. 

- http://prntscr.com/c4ky36 - caused WITH random light updates + FAWE. The glitch occours even in chunks that I never used the plugin (I teleported to a 8000 blocks away coordinate, and there's many lighting problems under trees, caves and whatever). I removed the plugin and it's back to normal. It updates after placing blocks, but when you relog it glitches again.
